nipype/algorithms/confounds.py

```diff
@@ -681,11 +681,12 @@ def compute_dvars(in_file, in_mask, remove_zerovariance=False,
     # standardization
     dvars_stdz = dvars_nstd / diff_sd_mean
 
-    with warnings.catch_warnings(): # catch, e.g., divide by zero errors
+    with warnings.catch_warnings():  # catch, e.g., divide by zero errors
         warnings.filterwarnings('error')
 
         # voxelwise standardization
-        diff_vx_stdz = np.square(func_diff) / np.array([diff_sdhat] * func_diff.shape[-1]).T
+        diff_vx_stdz = np.square(
```

Comment:
```**2``` would be more readable to me than ```np.square```, but it is just cosmetic. Maybe you prefer np.square :)